content/ko/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -0,0 +1,399 @@
+많은 것을 통신하는 두 개의 서로 다른 서비스의 파드를 같이 배치할 수 있다.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## 노드 셀렉터(nodeSelector)
+
+`nodeSelector` 는 가장 간단하고 권장하는 노드 선택 제약 조건의 형태이다.
+`nodeSelector` 는 PodSpec의 필드이다. 이는 키-값 쌍의 매핑으로 지정한다. 파드가 노드에서 동작할 수 있으려면,
+노드는 키-값의 쌍으로 표시되는 레이블을 각자 가지고 있어야 한다(이는 추가 레이블을 가지고 있을 수 있다).
+대부분의 일반적인 사용은 하나의 키-값 쌍이다.
+
+`nodeSelector` 를 어떻게 사용하는지 예시를 통해 알아보도록 하자.
+
+### 0 단계: 사전 준비
+
+이 예시는 쿠버네티스 파드에에 대한 기본적인 이해를 하고 있고 [쿠버네티스 클러스터가 설정](/ko/docs/setup/)되어 있다고 가정한다.
+
+### 1 단계: 노드에 레이블 붙이기
+
+`kubectl get nodes` 를 실행해서 클러스터 노드 이름을 가져온다. 이 중에 레이블을 추가하기 원하는 것 하나를 선택한 다음에 `kubectl label nodes <노드 이름> <레이블 키>=<레이블 값>` 을 실행해서 선택한 노드에 레이블을 추가한다. 예를 들어 노드의 이름이 'kubernetes-foo-node-1.c.a-robinson.internal' 이고, 원하는 레이블이 'disktype=ssd' 라면, `kubectl label nodes ubernetes-foo-node-1.c.a-robinson.internal disktype=ssd` 를 실행한다.
+
+`kubectl get nodes --show-labels` 를 다시 실행해서 노드가 현재 가진 레이블을 확인하여, 이 작업을 검증할 수 있다. 또한 `kubectl describe node "노드 이름"` 을 사용해서 노드에 주어진 레이블의 전체 목록을 확인할 수 있다.
+
+### 2 단계: 파드 설정에 nodeSelector 필드 추가하기
+
+실행하고자 하는 파드의 설정 파일을 가져오고, 이처럼 nodeSelector 섹션을 추가한다. 예를 들어 이것이 파드 설정이라면:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+  labels:
+    env: test
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+```
+
+이 다음에 nodeSelector 를 다음과 같이 추가한다:
+
+{{< codenew file="pods/pod-nginx.yaml" >}}
+
+그런 다음에 `kubectl apply -f https://k8s.io/examples/pods/pod-nginx.yaml` 을
+실행하면, 레이블이 붙여진 노드에 파드가 스케줄 된다.
+`kubectl get pods -o wide` 를 실행해서 파드가 할당된
+"NODE" 를 보면 작동하는지 검증할 수 있다.
+
+## 넘어가기 전에: 내장 노드 레이블들 {#built-in-node-labels}
+
+[붙인](#1-단계-노드에-레이블-붙이기) 레이블뿐만 아니라, 노드에는
+표준 레이블 셋이 미리 채워져 있다. 이 레이블들은 다음과 같다.
+
+* [`kubernetes.io/hostname`](/docs/reference/kubernetes-api/labels-annotations-taints/#kubernetes-io-hostname)
+* [`failure-domain.beta.kubernetes.io/zone`](/docs/reference/kubernetes-api/labels-annotations-taints/#failure-domainbetakubernetesiozone)
+* [`failure-domain.beta.kubernetes.io/region`](/docs/reference/kubernetes-api/labels-annotations-taints/#failure-domainbetakubernetesioregion)
+* [`topology.kubernetes.io/zone`](/docs/reference/kubernetes-api/labels-annotations-taints/#topologykubernetesiozone)
+* [`topology.kubernetes.io/region`](/docs/reference/kubernetes-api/labels-annotations-taints/#topologykubernetesiozone)
+* [`beta.kubernetes.io/instance-type`](/docs/reference/kubernetes-api/labels-annotations-taints/#beta-kubernetes-io-instance-type)
+* [`node.kubernetes.io/instance-type`](/docs/reference/kubernetes-api/labels-annotations-taints/#nodekubernetesioinstance-type)
+* [`kubernetes.io/os`](/docs/reference/kubernetes-api/labels-annotations-taints/#kubernetes-io-os)
+* [`kubernetes.io/arch`](/docs/reference/kubernetes-api/labels-annotations-taints/#kubernetes-io-arch)
+
+{{< note >}}
+이 레이블들의 값은 클라우드 공급자에 따라 다르고 신뢰성이 보장되지 않는다.
+예를 들어 `kubernetes.io/hostname` 은 어떤 환경에서는 노드 이름과 같지만,
+다른 환경에서는 다른 값일 수 있다.
+{{< /note >}}
+
+## 노드 격리(isolation)/제한(restriction)

```

Comment:
I did wonder whether this would work better as:
```suggestion
## 노드 격리(isolation)/제한(restriction) {#노드-격리-제한}
```
(and similar below)

What do you think @KimMJ?

Reply:
@sftim 
Do not add identifiers that are not currently in the text.
The addition of arbitrary identifiers is also a very important issue that will cause a full revision to other documents that reference this document.

Reply:
> not currently in the text.

My thinking here: as this is a new page, such a proposed change could not break any inbound link.